suite-common/message-system/config/config.v1.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Should not be merged with this. Add to temp commit, which will be failing CI. Or better, just put the message into the desc of PR, so that everyone who wants to test it has to run it on their own